src/main/java/Bob/task/Deadline.java

```diff
@@ -0,0 +1,43 @@
+package bob.task;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Locale;
+
+public class Deadline extends Task {
+
+    private LocalDateTime by;
+
+    /**
+     * Converts a LocalDateTime object to a string.
+     *
+     * @param dateTime
+     * @return String
+     */
+    private static String dateTimeToString(LocalDateTime dateTime) {
+        // Correct format: "MMM dd yyyy" e.g., (Oct 15 2019)
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("MMM dd yyyy HH:mm", Locale.ENGLISH);
+        return dateTime.format(formatter);
+    }
```

Comment:
Currently, this method is also defined in `Event`, which is fine. But personally I foresee that this would be reused quite a bit. Is there any reason why you wouldn't want to refactor this (together with the complement function in `Parser`) to another file?